<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1365</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,0,0">
    <item>
     <layout class="QVBoxLayout" name="DashboardLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="infoText">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>9</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1365</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="actionExporter"/>
    <addaction name="separator"/>
    <addaction name="actionFermer"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEditer">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAjouter_ligne"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionMode_nuit"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEditer"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="text">
    <string>Save as..</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAjouter_ligne">
   <property name="text">
    <string>Add row</string>
   </property>
  </action>
  <action name="actionSupprimer_ligne">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionCopier_ligne">
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="text">
    <string>Coller</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionMode_nuit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark mode</string>
   </property>
  </action>
  <action name="actionExporter">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
